packages/lodestar-cli/src/cmds/beacon/handler.ts

```diff
@@ -58,6 +59,10 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
   abortController.signal.addEventListener("abort", dbClose, {once: true});
   await db.start();
 
+  if (args.testnet && !args.genesisStateFile) {
```

Comment:
Would it make more sense to move this logic into `src/cmds/beacon/initBeaconState.ts`?

Reply:
yeah i think so.  just pushed an update for this.